resources/views/livewire/admin/roles/index.blade.php

```diff
@@ -0,0 +1,98 @@
+            <!-- Botón de nuevo -->
+            <div>
+                <x-jet-secondary-button wire:click="create()" class="border-green-300 text-green-700 hover:text-green-500 active:text-green-800 active:bg-green-50">
+                    <x-icon.plus solid alt="sm" class="inline-block h-5 w-5"/>
+                    Nuevo rol
+                </x-jet-secondary-button>
+            </div>
+
+            <!-- Opciones de tabla -->
+            <div class="md:flex md:justify-between space-y-2 md:space-y-0">
+                <!-- Parte izquierda -->
+                <div class="md:w-1/2 md:flex space-y-2 md:space-y-0 md:space-x-2">
+                    <!-- Barra de búsqueda -->
+                    <x-input.icon wire:model="search" class="w-full" type="text" placeholder="Buscar rol...">
+                        <x-icon.search solid class="h-5 w-5 text-gray-400"/>
+                    </x-input.icon>
+                </div>
+
+                <!-- Parte derecha -->
+                <div class="md:flex md:items-center space-y-2 md:space-y-0 md:space-x-2">
+                    <!-- Selección de paginación -->
+                    <div>
+                        <x-input.select wire:model="perPage" class="block w-full">
+                            <option value="5">5</option>
+                            <option value="10">10</option>
+                            <option value="15">15</option>
+                            <option value="25">25</option>
+                        </x-input.select>
+                    </div>
+                </div>
+            </div>
+
+            <!-- Tabla -->
+            <div class="flex flex-col space-y-2">
+                <x-table>
+                    <x-slot name="head">
+                        <x-table.header wire:click="sortBy('name')" sortable :direction="$sortField === 'name' ? $sortDirection : null">
+                            nombre
+                        </x-table.header>
+                        <x-table.header>acciones</x-table.header>
+                    </x-slot>
+
+                    @forelse($roles as $r)
+                        @unless($r->name === 'Super admin' && auth()->user()->getRoleNames()->first() !== 'Super admin')
+                            <tr wire:key="role-{{ $r->id }}" wire:loading.class.delay="opacity-50">
+                                <x-table.cell>{{ $r->name }}</x-table.cell>
+                                <x-table.cell>
+                                    @unless($r->name === 'Super admin')
+                                        <button wire:click="edit({{ $r->id }})" type="button" class="text-amber-600 hover:text-amber-900">
+                                            <x-icon.pencil alt class="h-6 w-6"/>
+                                        </button>
+                                    @endunless
+
+                                    @unless(in_array($r->name, ['Super admin', 'Administrador', 'Participante', 'Instructor']))
```

Comment:
Fa aquí estableces que Participante e instructor pueden eliminar?, ¿no deberían no es así?, unicamente el admin o el super admin

Reply:
No, no hace eso. Lo que hace es que si el el **rol mostrado en la tabla** resulta ser 'Super admin', 'Administrador', 'Participante' o 'Instructor', no mostrará el botón de eliminar.

Reply:
Pero aplica solo para los primeros 3 verdad, los creados, todos o solo el admin puede elminar 
![image](https://user-images.githubusercontent.com/36526810/168457674-0547971d-8c7b-4c58-8977-a80abfa67b28.png)
 (imagen con usuario creado)

Reply:
Más especificamente: Busca el nombre del rol actual _(que se mostrará como una fila en la tabla, es decir, la iteración actual del `for`)_ dentro de el arreglo `['Super admin', 'Administrador', 'Participante', 'Instructor']`. Si coincide con alguno de estos, pues no muestra el botón de eliminar.

Puedes probarlo entrando con diferentes roles.

Reply:
> Pero aplica solo para los primeros 3 verdad, los creados, todos o solo el admin puede elminar

Así es, solo para los roles bases

Reply:
Si lo probé con los tipos de roles, pero no comprendia porque no se eliminan los primero y los nuevos creados sí, pero viendo y analizando, si es necesario que no se eliminen los de base, pero eso abree algo, espera te hago comentario ahí

Reply:
> Si lo probé con los tipos de roles, pero no comprendia porque no se eliminan los primero y los nuevos creados sí, pero viendo y analizando, si es necesario que no se eliminen los de base, pero eso abree algo, espera te hago comentario ahí

Oki doki, espero